proto/frequenz/api/common/v1/pagination.proto

```diff
@@ -0,0 +1,34 @@
+// Parameters for pagination.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.pagination;
+
+// A message defining parameters for paginating list requests.
+message PaginationParams {
+  // [Optional]
+  // The maximum number of results to be returned per request.
+  // Defaults to 50 if omitted.
+  uint32 page_size = 1;
+
+  // [Optional]
+  // The token identifying a specific page of the list results.
+  string page_token = 2;
+}
+
+// A message providing metadata about paginated list results.
```

Comment:
It would be nice if there is a code example on how to use this message. A simple RPC would be enough as an example.

Reply:
You mean as in how to utilize this response message to receive the next page from the API?